cmd/deploy.go

```diff
@@ -99,9 +99,16 @@ func newDeployCmd(rootConfig *RootCommandConfig) *cobra.Command {
 
 	var deployCmd = &cobra.Command{
 		Use:   "deploy",
-		Short: "Build and deploy your Appsody project to your Kubernetes cluster",
-		Long: `This command extracts the code from your project, builds a local Docker image for deployment,
-generates a deployment manifest (yaml) file if one is not present, and uses it to deploy your image to a Kubernetes cluster, either via the Appsody operator or as a Knative service.`,
+		Short: "Build and deploy your Appsody project to Kubernetes.",
+		Long: `Build and deploy a local container image of your Appsody project to your Kubernetes cluster. 
+		
+The stack, along with your Appsody project, is extracted to a local directory before the container image of your Appsody project is built for deployment.
+The command generates a deployment manifest file, "app-deploy.yaml", if one is not present. The CLI uses this file to deploy your image to your Kubernetes cluster, either via the Appsody operator or as a Knative service.`,
```

Comment:
I wonder if we should change the wording about Knative Service here, as of milestone 10 we will always use the appsody operator.   The --knative option determines if the appsody-operator deploys the application as a Knative Service.

Reply:
@kewegner How about "The CLI uses this file to deploy your image to your Kubernetes cluster via the Appsody operator, and optionally as a Knative service if you specify the '--knative' flag."?

Reply:
Yes I think that wording is OK.